src/test/java/spiderman/TodoTest.java

```diff
@@ -0,0 +1,28 @@
+package spiderman;  //same package as the class being tested
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I think explicit imports would make the dependencies clearer - maybe this was automatically done by the IDE?